include/sybdb.h

```diff
@@ -1278,7 +1278,7 @@ RETCODE dbsetlversion (LOGINREC * login, BYTE version);
 #define DBSETLPACKET(x,y)	dbsetllong((x), (y), DBSETPACKET)
 #define dbsetlpacket(x,y)	dbsetllong((x), (y), DBSETPACKET)
 #define DBSETENCRYPT		12
-#define DBSETLENCRYPT(x,y)	dbsetlbool((x), (y), DBSETENCRYPT)
+#define DBSETLENCRYPT(x,y)	dbsetlname((x), (y), DBSETENCRYPT)
```

Comment:
this breaks both ABI and API compatibility

Reply:
Any suggestions on how to deal with this problem? `DBSETENCRYPT` in `dbsetlbool` is unimplemented and marked as TODO.

Reply:
From the API prospective a line like `DBSETLENCRYPT(login, TRUE/FALSE)` should continue to compile. From the ABI prospective `dbsetlbool(login, TRUE/FALSE, 12)` should continue to set something for encryption. I would personally add a new setting, it could be something like
```c
#define DBSETENCRYPTION  1005
#define DBSETLENCRYPTION(x, y)  dbsetlname((x), (y), DBSETENCRYPTION)
```
`ENCRYPTION` is just a suggestion. The `1005` was the next FreeTDS extension from `include/sybdb.h`.